capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.google.univiz.api;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+
+class SearchResourceImpl implements SearchResource {
+  private List<SuggestionData> collegeSuggestions;
+
+  @Inject
+  public void Restaurant(collegeSuggestions) {
+    this.collegeSuggestions = collegeSuggestions;
```

Comment:
```suggestion
  public SearchResourceImpl(List<SuggestionData> collegeSuggestions) {
      this.collegeSuggestions = new ArrayList<>();
      this.collegeSuggestions.addAll(collegeSuggestions);
```
Few things here:

- This looks like a constructor, and if that assumption is correct, then the name and return type were off.
- The assignment here is a shallow assignment, which is fine sometimes, but it also means anyone/any class with the reference to `collegeSuggestions` will be able to manipulate the instance's copy of `collegeSuggestions` even after the instance has been created. Given that `collegeSuggestions` is a private member, it's probably a good idea to make sure it only gets manipulated by `SearchResourceImpl`'s methods.
- To get Guice to know how to inject `collegeSuggestions` remember to write a subclass of `AbstractModule`.